lib/domain/service/my_user.dart

```diff
@@ -57,6 +57,7 @@ class MyUserService extends DisposableService {
       onPasswordUpdated: _onPasswordUpdated,
       onUserDeleted: _onUserDeleted,
     );
+    _userRepo.myUser.value = _auth.user;
     super.onInit();
```

Comment:
1) This only changes the `Rx` value of `MyUser` repository. Instead `MyUser` must be persisted, which only may happen, if you put the model into Hive storage. Imagine you are signing in into an account, and suddenly a network error happens, causing `MyUserRepository` to never subscribe to `myUserEvents`. Refreshing the application will cause the data you put there to be lost.
Also we have a common rule in the project, which I'll try to add to the Contribution Guide: `Rx` values in repositories mustn't be updated from any place outside the repository itself. Instead the repository owning those entities must update them itself, or via Hive local subscription. This encapsulates the assignment logic inside the repository. This helps keeping nice and maintainable flow: you put the data to Hive, Hive then notifies the repository about item being updated, and repository updates the `Rx` field itself using the stored item. Is it clear to you, or perhaps you need more explanation?

2) By putting this code there, in `onInit` of `MyUserService`, you aren't really connected to the fact of `AuthRepository` retrieving the `MyUser`. It just "happens" to be there at this moment. This will surely break, for example, if we'll decide to bind the lifecycle of `MyUserService` to the application itself. `onInit` would only happen once this way.